clients/src/main/java/org/apache/kafka/clients/producer/internals/BufferPool.java

```diff
@@ -43,6 +43,8 @@
  */
 public class BufferPool {
 
+    static final String WAIT_TIME_SENSOR_NAME = "bufferpool-wait-time";
```

Comment:
There is a mock that refers to this variable otherwise I would not have bothered to put this into a static final variable.